includes/admin/class-wc-admin-addons.php

```diff
@@ -506,4 +508,29 @@ public static function install_woocommerce_services_addon() {
 		wp_safe_redirect( remove_query_arg( array( 'install-addon', '_wpnonce' ) ) );
 		exit;
 	}
+
+	/**
+	 * Should an extension be shown on the featured page.
+	 *
+	 * @param object $item
+	 *
+	 * @return boolean
+	 */
+	public static function show_extension( $item ) {
+		$location = WC()->countries->get_base_country();
+		$show = true;
+		if ( isset( $item->geowhitelist ) && ! in_array( $location, $item->geowhitelist ) ) {
+			$show = false;
```

Comment:
If these conditions all return false, why not return here and dump the `$show` variable? If it gets to the end of the func, return true.